deployment/GI02HYPM_Deploy.csv

```diff
@@ -76,12 +76,12 @@ AR69-01,,AR76-03,,GI02HYPM-WFP02-04-CTDPFL000,9,1,2022-07-01T12:49:00,2023-09-06
 AR69-01,,AR76-03,,GI02HYPM-MPM01-02-ZPLSGA010,9,1,2022-07-01T12:49:00,2023-09-06T17:43:00,CGMGI-02HYPM-00009,,CGINS-ZPLSGA-55111,59.969927,-39.489243,,150,2676,downward-looking,,D
-AR76-03,,,,GI02HYPM-RIM01-00-SIOENG000,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,,CGCON-SIOCON-13278,59.973718,-39.525233,,2570,2661,,,D
-AR76-03,,,,GI02HYPM-RIM01-02-CTDMOG039,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,,CGINS-CTDMOG-12609,59.973718,-39.525233,,155,2661,,,D
-AR76-03,,,,GI02HYPM-WFP02-00-WFPENG000,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,CGVEH-GWFPGX-45201,CGCON-GWFPGX-45201,59.973718,-39.525233,,2560,2661,,,D
-AR76-03,,,,GI02HYPM-WFP02-01-FLORDL000,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,CGVEH-GWFPGX-45201,CGINS-FLORDL-04013,59.973718,-39.525233,,2560,2661,,,D
-AR76-03,,,,GI02HYPM-WFP02-03-DOSTAL000,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,CGVEH-GWFPGX-45201,CGINS-DOSTAL-02198,59.973718,-39.525233,,2560,2661,,,D
-AR76-03,,,,GI02HYPM-WFP02-04-CTDPFL000,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,CGVEH-GWFPGX-45201,CGINS-CTDPFL-00150,59.973718,-39.525233,,2560,2661,,,D
-AR76-03,,,,GI02HYPM-WFP02-05-VEL3DL000,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,CGVEH-GWFPGX-45201,CGINS-VEL3DL-01256,59.973718,-39.525233,,2560,2661,,,D
-AR76-03,,,,GI02HYPM-MPM01-02-ZPLSGA009,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,,CGINS-ZPLSGA-55092,59.973718,-39.525233,,150,2661,upward-looking,,D
-AR76-03,,,,GI02HYPM-MPM01-02-ZPLSGA010,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,,CGINS-ZPLSGA-55093,59.973718,-39.525233,,150,2661,downward-looking,,D
+AR76-03,,,,GI02HYPM-RIM01-00-SIOENG000,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,,CGCON-SIOCON-13278,59.973718,-39.525223,,2570,2661,,,D
+AR76-03,,,,GI02HYPM-RIM01-02-CTDMOG039,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,,CGINS-CTDMOG-12609,59.973718,-39.525223,,155,2661,,,D
+AR76-03,,,,GI02HYPM-WFP02-00-WFPENG000,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,CGVEH-GWFPGX-45201,CGCON-GWFPGX-45201,59.973718,-39.525223,,2560,2661,,,D
+AR76-03,,,,GI02HYPM-WFP02-01-FLORDL000,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,CGVEH-GWFPGX-45201,CGINS-FLORDL-04013,59.973718,-39.525223,,2560,2661,,,D
+AR76-03,,,,GI02HYPM-WFP02-03-DOSTAL000,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,CGVEH-GWFPGX-45201,CGINS-DOSTAL-02198,59.973718,-39.525223,,2560,2661,,,D
+AR76-03,,,,GI02HYPM-WFP02-04-CTDPFL000,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,CGVEH-GWFPGX-45201,CGINS-CTDPFL-00150,59.973718,-39.525223,,2560,2661,,,D
+AR76-03,,,,GI02HYPM-WFP02-05-VEL3DL000,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,CGVEH-GWFPGX-45201,CGINS-VEL3DL-01256,59.973718,-39.525223,,2560,2661,,,D
+AR76-03,,,,GI02HYPM-MPM01-02-ZPLSGA009,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,,CGINS-ZPLSGA-55092,59.973718,-39.525223,,150,2661,upward-looking,,D
+AR76-03,,,,GI02HYPM-MPM01-02-ZPLSGA010,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,,CGINS-ZPLSGA-55093,59.973718,-39.525223,,150,2661,downward-looking,,D
```

Comment:
Looks good. Decimal degrees match converted longitude from config sheet.

Reply:
my calculated longitude is -39.52523 from 39 31.5138 (value in config sheet and qlr).